stylus/components/modals.styl

```diff
@@ -8,91 +10,94 @@
 \*------------------------------------*/
 
 $modals
-    modal-width = em(480px)
-
-    .coz-overlay
-        z-index     $modal-index
-        position    fixed
-        top         0
-        left        0
-        height      100%
-        width       100%
-        background  rgba(charcoal-grey, .5)
-        visibility  visible
-        transition  opacity .3s, visibility 0s ease-out
-
-        &.coz-overlay--hidden
-            opacity     0
-            visibility  hidden
-            transition  opacity .3s 0s, visibility 0s ease-in .3s
-
+    modal-width = 30rem
 
     .coz-modal-wrapper
         position          fixed
         top               0
         left              0
         display           flex
         flex-direction    row
+        align-items       start
         box-sizing        border-box
         width             100vw
         height            100vh
         overflow-y        scroll
-        padding           (16/basefont)em
+        padding           1rem
 
     .coz-modal
         position          relative
-        margin            5vh auto auto
-        border-radius     em(8px)
-        max-width         90%
+        margin            0 auto
```

Comment:
So now modals can take all the viewport's available space? Isn't it nice to have some padding/margin to lighten the UI?

Reply:
The padding is provided by `.coz-modal-wrapper` just above :)

Reply:
Ah missed it. My bad.